buf/registry/module/v1beta1/label_service.proto

```diff
@@ -74,10 +74,10 @@ message ListLabelsRequest {
   // The list order.
   enum Order {
     ORDER_UNSPECIFIED = 0;
-    // Order by create_time oldest to newest.
-    ORDER_CREATE_TIME_ASC = 1;
     // Order by create_time newest to oldest.
-    ORDER_CREATE_TIME_DESC = 2;
+    ORDER_CREATE_TIME_DESC = 1;
```

Comment:
Switched this because it seems like we are consistent about the default being first